extensions/wikia/ImageServing/api/WikiaApiImageServing.php

```diff
@@ -23,7 +23,7 @@ public function __construct( $main, $action ) {
 	 * See functions below for expected URL params
 	 */
 	public function execute() {
-		global $wgRequest, $wgCacheBuster;
+		global $wgRequest, $wgStyleVersion;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgRequest" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app